bahmni-proxy/README.md

```diff
@@ -12,4 +12,16 @@ This directory contains resources for building Docker image for Bahmni Proxy Con
      > export BAHMNI_VERSION=0.94 
 
      > export GITHUB_RUN_NUMBER=1
-4. Run the build script `./bahmni-proxy/scripts/docker_build.sh`
\ No newline at end of file
+4. Run the build script `./bahmni-proxy/scripts/docker_build.sh`
+
+## Integrate [speech-assistant-package](https://github.com/Bahmni/speech-assistant-package)
+  * [speech-recognition-open-api-proxy](https://github.com/Open-Speech-EkStep/speech-recognition-open-api-proxy) is using websocket protocol
+  * To pass a request from bahmni-proxy to speech-recognition-open-api-proxy, put below lines in *<VirtualHost \*:443>* tag:
+    ```
+    RewriteEngine on 
+    RewriteCond %{HTTP:Upgrade} websocket [NC]
+    RewriteCond %{HTTP:Connection} upgrade [NC]
+    Header set Content-Security-Policy upgrade-insecure-requests
+    RewriteRule ^/?(.*) "ws://vakyansh-proxy:9009/$1" [P,L]
+    ```
```

Comment:
this readme is not making sense... what do you expect developers to do?

Reply:
updated the readme file